libraries/base/GHC/IO/Encoding/Iconv.hs

```diff
@@ -58,12 +59,12 @@ iconv_trace s
 
 {-# NOINLINE localeEncodingName #-}
```

Comment:
@puffnfresh one question, why we need `NOINLINE` pragma here? i mean, we don't use something like `inlinePerformIO`, so what exactly the reason that we should put it here?